utils_traintest.py

```diff
@@ -0,0 +1,59 @@
+from dataloader import DigitDataset
+from utils import reshape, drawimg, tensor_to_pltimg, multidrawimg, train, test
+import torch
+from torch import nn
+from torch.utils.data import Dataset
+from torch.utils.data import DataLoader
+import os
+os.environ['KMP_DUPLICATE_LIB_OK']='True'
+
+import csv
```

Comment:
```suggestion
import os
import csv
os.environ['KMP_DUPLICATE_LIB_OK']='True'

import torch
from torch import nn
from torch.utils.data import Dataset
from torch.utils.data import DataLoader

from dataloader import DigitDataset
import utils as ut
```

그리고 뒤에 reshape이나 여러 utils에서 온 함수들 ut.reshape으로 사용하기